### DISCUSSION TITLE
Additional Endpoint configuration not working for Logs_config

### DISCUSSION THREAD

**Discussion by jeyakumar8** [DataDog/datadog-agent#19296] [General]
Hi,

Problem statement:
I have my datadog-agent running in Kubernetes cluster as a Daemonset. I'm using Additional endpoints feature of datadog-agent to send logs to an alternate pipeline(vector.dev) which will then process it and stores in prometheus.

I have configured the logs_config.additonal_endpoints with vector agent endpoint.

'''
logs_config:
      use_http: true
      additional_endpoints:
      - api_key: <api_key>
        Host: "vector.vector.svc.cluster.local"
        Port: "9999"
        is_reliable: true
'''

But when i start the agent the logs are not flowing to the additional endpoint. But it is working properly for the main intake endpoint.

I did a manual curl for api/v2/logs, it is working as expected, only the agent is not able to detect the endpoint.

Logs:
There is no Error logs in the agent.log.

Could someone help me resolve this?

Thanks in Advance

**Comment by tbenade** [comment]
Hi @jeyakumar8 did you get anywhere with this. I too am about to embark on an effort to send telemetry to Vector before shipping on to Datadog?
